index.html

```diff
@@ -4,23 +4,27 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="./assets/css/style.css">
+    <link rel="stylesheet" href="./assets/css/querie.css">
     <title>BooksCollection</title>
 </head>
 <body>
     <header>
-        <h1>All awesome Books</h1>
+        <h1 id="top-title0">All awesome Books</h1>
     </header>
     <main class="container">
```

Comment:
### [Optional]
It would be nice to reduce the length of this div `container` to `width:50%`, I think that way it will really fit in with the given layout.